.github/workflows/update-antora-ui-spring.yml

```diff
@@ -0,0 +1,35 @@
+name: Update Antora UI Spring
+
+on:
+  schedule:
+    - cron: '0 10 * * *' # Once per day at 10am UTC
+  workflow_dispatch:
+
+permissions:
+  pull-requests: write
+  issues: write
+  contents: write
+
+jobs:
+  update-antora-ui-spring:
+    runs-on: ubuntu-latest
+    name: Update on Supported Branches
+    strategy:
+      matrix:
+        branch: [ '5.3.x', '6.0.x', '6.1.x', 'main' ]
```

Comment:
@marcusdacoregio Thanks for this PR. We don't use Antora on `5.3.x`, can I refine this PR to remove this branch?

Reply:
Sure, feel free to make any changes